### ISSUE TITLE
Bundles filtered log is hard to read

### ISSUE THREAD

**Issue by jmrodri** [automationbroker/bundle-lib#147]
Feels like the log statement shouldn't be all on one line. it's hard to read:

```
time="2018-07-23T21:27:57Z" level=info msg="Bundles filtered by white/blacklist filter:-> ansibleplaybookbundle/apb-tools-> ansibleplaybookbundle/origin-ansible-service-broker-> ansibleplaybookbundle/hello-world-> ansibleplaybookbundle/mediawiki123-> ansibleplaybookb     undle/ansible-service-broker-> ansibleplaybookbundle/apb-base-> ansibleplaybookbundle/origin-> ansibleplaybookbundle/py-zip-demo-> ansibleplaybookbundle/asb-installer-> ansibleplaybookbundle/photo-album-demo-app-> ansibleplaybookbundle/helm-bundle-base-> ansibleplayb     ookbundle/deploy-broker-> ansibleplaybookbundle/photo-album-demo-api-> ansibleplaybookbundle/apb-assets-base-> ansibleplaybookbundle/manageiq-apb-runner-> ansibleplaybookbundle/vnc-client-> ansibleplaybookbundle/mediawiki-> ansibleplaybookbundle/origin-service-catalo     g-> ansibleplaybookbundle/origin-recycler-> ansibleplaybookbundle/vnc-desktop-> ansibleplaybookbundle/origin-deployer-> ansibleplaybookbundle/origin-docker-registry-> ansibleplaybookbundle/origin-haproxy-router-> ansibleplaybookbundle/origin-pod-> ansibleplaybookbund     le/origin-sti-builder-> ansibleplaybookbundle/kubevirt-ansible"
```
The `->` should be in front of the apbs, but it looks like it is after.

```golang
     if len(filteredNames) != 0 {
         var buffer bytes.Buffer
         buffer.WriteString("Bundles filtered by white/blacklist filter:")
         for _, name := range filteredNames {
             buffer.WriteString(fmt.Sprintf("-> %s", name))
         }
         log.Infof(buffer.String())
     }
```

**Comment by jmrodri** [comment]
If we keep it on a single line, I suggest we replace `-> ` with maybe ` |` 

```
time="2018-07-23T21:27:57Z" level=info msg="Bundles filtered by white/blacklist filter: |ansibleplaybookbundle/apb-tools |ansibleplaybookbundle/origin-ansible-service-broker |ansibleplaybookbundle/hello-world |ansibleplaybookbundle/mediawiki123 |ansibleplaybookbundle/ansible-service-broker |ansibleplaybookbundle/apb-base |ansibleplaybookbundle/origin |ansibleplaybookbundle/py-zip-demo |ansibleplaybookbundle/asb-installer |ansibleplaybookbundle/photo-album-demo-app |ansibleplaybookbundle/helm-bundle-base |ansibleplayb  ookbundle/deploy-broker |ansibleplaybookbundle/photo-album-demo-api |ansibleplaybookbundle/apb-assets-base |ansibleplaybookbundle/manageiq-apb-runner |ansibleplaybookbundle/vnc-client |ansibleplaybookbundle/mediawiki |ansibleplaybookbundle/origin-service-catalog |ansibleplaybookbundle/origin-recycler |ansibleplaybookbundle/vnc-desktop |ansibleplaybookbundle/origin-deployer |ansibleplaybookbundle/origin-docker-registry |ansibleplaybookbundle/origin-haproxy-router |ansibleplaybookbundle/origin-pod |ansibleplaybookbundle/origin-sti-builder |ansibleplaybookbundle/kubevirt-ansible"
```

**Comment by jmrodri** [comment]
I think I'd prefer one per line instead:

```
time="2018-07-23T21:27:57Z" level=info msg="Bundles filtered by white/blacklist filter:
    -> ansibleplaybookbundle/apb-tools
    -> ansibleplaybookbundle/origin-ansible-service-broker
    -> ansibleplaybookbundle/hello-world
    -> ansibleplaybookbundle/mediawiki123
    -> ansibleplaybookbundle/ansible-service-broker
    -> ansibleplaybookbundle/apb-base
    -> ansibleplaybookbundle/origin
    -> ansibleplaybookbundle/py-zip-demo
    -> ansibleplaybookbundle/asb-installer
    -> ansibleplaybookbundle/photo-album-demo-app
    -> ansibleplaybookbundle/helm-bundle-base
    -> ansibleplaybookbundle/deploy-broker
    -> ansibleplaybookbundle/photo-album-demo-api
    -> ansibleplaybookbundle/apb-assets-base
    -> ansibleplaybookbundle/manageiq-apb-runner
    -> ansibleplaybookbundle/vnc-client
    -> ansibleplaybookbundle/mediawiki
    -> ansibleplaybookbundle/origin-service-catalog
    -> ansibleplaybookbundle/origin-recycler
    -> ansibleplaybookbundle/vnc-desktop
    -> ansibleplaybookbundle/origin-deployer
    -> ansibleplaybookbundle/origin-docker-registry
    -> ansibleplaybookbundle/origin-haproxy-router
    -> ansibleplaybookbundle/origin-pod
    -> ansibleplaybookbundle/origin-sti-builder
    -> ansibleplaybookbundle/kubevirt-ansible"
```

**Comment by jmrodri** [comment]
Or better yet one per line
```
time="2018-07-23T21:27:57Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/apb-tools
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-ansible-service-broker
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/hello-world
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/mediawiki123
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/ansible-service-broker
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/apb-base
time="2018-07-23T21:27:58Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/py-zip-demo
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/asb-installer
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/photo-album-demo-app
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/helm-bundle-base
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/deploy-broker
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/photo-album-demo-api
time="2018-07-23T21:27:59Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/apb-assets-base
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/manageiq-apb-runner
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/vnc-client
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/mediawiki
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-service-catalog
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-recycler
time="2018-07-23T21:27:60Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/vnc-desktop
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-deployer
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-docker-registry
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-haproxy-router
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-pod
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/origin-sti-builder
time="2018-07-23T21:27:61Z" level=info msg="Bundles filtered by white/blacklist filter: -> ansibleplaybookbundle/kubevirt-ansible"
```

**Comment by jmrodri** [comment]
Let's make it look like we did with **Bundles passing white/blaclist filter:"
```golang
if len(validNames) != 0 {                                
    log.Debugf("Bundles passing white/blacklist filter:")
    for _, name := range validNames {                    
        log.Debugf("-> %s", name)                        
    }                                                    
}                                                        
```

```
time="2018-07-23T21:28:08Z" level=debug msg="Bundles passing white/blacklist filter:"
 465 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/php"                  
 466 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/postgresql"           
 467 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/python"               
 468 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/httpd"                
 469 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/mysql"                
 470 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/nginx"                
 471 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/nodejs"               
 472 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/perl"                 
 473 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/redis"                
 474 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/ruby"                 
 475 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/jenkins"              
 476 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/mariadb"              
 477 time="2018-07-23T21:28:08Z" level=debug msg="-> openshift/mongodb"              
```

**Comment by jmrodri** [comment]
Fixed by PR #149
